charts/flink-tools/values.yaml

```diff
@@ -22,6 +22,8 @@ localStorage:
   replicas: 2
   size: "5G"
 #mainClass: Name of main class. REQUIRED.
+enableExtraVolume: false
```

Comment:
This flag will be unneeded with above change.

Reply:
removed